docs/maui/converters/is-in-range-converter.md

```diff
@@ -0,0 +1,120 @@
+---
+title: IsInRangeConverter - .NET MAUI Community Toolkit
+author: GeorgeLeithead
+description: "The IsInRangeConverter is a one-way converter that takes an incoming value implementing IComparable, and a minimum and maximum value, and returns the result of the value being between the minimum and maximum values."
+ms.date: 11/26/2022
+---
+
+# IsInRangeConverter
+
+The `IsInRangeConverter` is a one-way converter that takes an incoming value implementing `IComparable`, and returns the result of the value being between the minimum and maximum values. The result will default to a `bool` if no objects were specified through the `TrueObject` and/or `FalseObject` properties. If values are assigned to the `TrueObject` and `FalseObject` properties, the IsInRangeConverter returns the respective object assigned.
+
+> [!NOTE]
+> Note that the either **both** the `TrueObject` and `FalseObject` should have a value defined or **neither** should.
+
+The `ConvertBack` method is not supported.
+
+[!INCLUDE [common converter properties](../includes/communitytoolkit-converter.md)]
+
+## Syntax
+
+### XAML
+
+The `IsInRangeConverter` can be used as follows in XAML:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.IsInRangeConverterPage">
+        <ResourceDictionary>
+            <x:String x:Key="MinValue">Hello</x:String>
+            <x:String x:Key="MaxValue">World</x:String>
+            <Style x:Key="MyValueIsInRangeTrue" TargetType="Label">
+                <Setter Property="BackgroundColor" Value="Green" />
+            </Style>
+            <Style x:Key="MyValueIsInRangeFalse" TargetType="Label">
+                <Setter Property="BackgroundColor" Value="Red" />
+            </Style>
+        </ResourceDictionary>
+
+    <Label Style="{Binding MyValue, Converter={mct:IsInRangeConverter TrueObject={StaticResource MyValueIsInRangeTrue}, FalseObject={StaticResource MyValueIsInRangeFalse}, MinValue={StaticResource MinValue}, MaxValue={StaticResource MaxValue}}}" Text="The background of this label will be green if MyValue is between 'Hello' and 'World', otherwise red." />
```

Comment:
typo in sample
```suggestion
    <Label Style="{Binding MyValue, Converter={toolkit:IsInRangeConverter TrueObject={StaticResource MyValueIsInRangeTrue}, FalseObject={StaticResource MyValueIsInRangeFalse}, MinValue={StaticResource MinValue}, MaxValue={StaticResource MaxValue}}}" Text="The background of this label will be green if MyValue is between 'Hello' and 'World', otherwise red." />
```